package.json

```diff
@@ -6,7 +6,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "nodemon",
-    "test": " nyc mocha -i -g 'Post-Deploy'",
+    "test": " nyc mocha -i -g 'Post-Deploy' --exit",
```

Comment:
running tests hangs for ~20s, will investigate

Reply:
probably a http2 hanging socket

Reply:
nope, I checked with `DEBUG=helix-fetch* npx nyc mocha ...`, and all connections are http1. stefan and I are currently investigating...

Reply:
see https://github.com/adobe/helix-fetch/issues/185